xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -496,11 +496,9 @@ void CGUIDialogVideoInfo::OnSearch(std::string& strSearch)
       if (iItem < 0)
         return;
 
-      CFileItem* pSelItem = new CFileItem(*items[iItem]);
+      CFileItemPtr pSelItem(new CFileItem(*items[iItem]));
 
-      OnSearchItemFound(pSelItem);
-
-      delete pSelItem;
+      OnSearchItemFound(pSelItem.get());
```

Comment:
And now the final solution: <code>OnSearchItemFound(items[iItem].get());</code>. I just realized that items is a list of CFileItemPtr...